CSS/about.css

```diff
@@ -12,17 +12,160 @@
     padding: 0;
     box-sizing: border-box;
 }
+
+body {
+    margin: 0;
+    padding: 0;
+    min-height: 100vh;
+    background-color: white;
+    font-family: "Dosis", sans-serif;
+    align-items: center;
+    justify-content: center;
+  }
+  
 #Mission-container {
     padding: 4rem 1rem;
     margin-top: 0px;
+    height: 40%;
     background-image: linear-gradient(to bottom, var(--COC-Blue) 0%, rgba(1, 27, 64, 0) 200%), url("https://b620782.smushcdn.com/620782/wp-content/uploads/2021/03/web-klassen-corp-036-850x458-1.jpg?lossy=1&strip=1&webp=1");
     background-size: cover;
-    color: white;
+    color:  rgba(228, 228, 228, 1);
+}
+.slide-container {
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 15px;
+}
+ 
+.biography {
+    position: relative;
+ 
+    margin: 0 auto;
+    background-color: #ffffff;
+    overflow: hidden;
+}
+      
+.biography .biography-inner {
+    position: relative;
+    width: 400%;
```

Comment:
This was a confusing element to read, try to add a short comment on why you are using `width: 400%`

This applies to other properties you think may need clarification.